### ISSUE TITLE
Elasticsearch log file name error when using x-pack

### ISSUE THREAD

**Issue by ofir123** [spujadas/elk-docker#94]
When installing x-pack on top of the default elasticsearch, everything works well except for the log file name part. Since the file name is derived from the cluster name, and cluster health checks cannot be made without proper credentials, the following error is raised:
```
touch: cannot touch '/var/log/elasticsearch/{"error":{"root_cause":[{"type":"security_exception","reason":"missingauthenticationtokenforRESTrequest[/_cat/health?h=cluster]","header":{"WWW-Authenticate":"Basicrealm=\"security\"charset=\"UTF-8\""}}],"type":"security_exception","reason":"missingauthenticationtokenforRESTrequest[/_cat/health?h=cluster]","header":{"WWW-Authenticate":"Basicrealm=\"security\"charset=\"UTF-8\""}},"status":401}.log': No such file or directory
```
A quick solution would be to accept CLUSTER_NAME as an environment variable, by simply deleting line 92:
```
CLUSTER_NAME=
```
Would love to hear your opinion about the fix.. (didn't open a PR since I wasn't sure if this line is there for some other reason)
Thanks!

**Comment by spujadas** [comment]
Yep, it's a caveat of X-Pack that I also picked up on when putting together the [elkx](https://github.com/spujadas/elkx-docker) image.
I was thinking about enabling a login and password to be passed through a couple of env vars so that the cluster name gets populated automatically, but your quick fix sounds fine to me for the time being.
I'd be more than happy to merge a PR if you want to open one.

**Comment by spujadas** [comment]
Fixed by #95
